dist/mapdc.js

```diff
@@ -26782,15 +26780,31 @@ function rasterLayerPolyMixin(_layer) {
       });
     }
 
-    var colorScaleName = getColorScaleName(layerName);
-    var scales = [{
-      name: colorScaleName,
-      type: "quantize",
-      domain: autocolors ? { data: getStatsLayerName(), fields: ["mincolor", "maxcolor"] } : state.encoding.color.domain,
-      range: colorRange,
-      nullValue: "rgba(214, 215, 214, 0.65)",
-      default: "rgba(214, 215, 214, 0.65)"
-    }];
+    var scales = [];
+    var fillColor = {};
+    var useColorScale = state.encoding.color.value === undefined;
+    if (useColorScale) {
+      var colorRange = state.encoding.color.range.map(function (c) {
```

Comment:
I guess we are confident that `state.encoding.color.range` will always be populated and will be an array.